parcels/particleset.py

```diff
@@ -1235,7 +1235,7 @@ def execute(
 
 def _warn_outputdt_release_desync(outputdt: float, release_times: Iterable[float]):
     """Gives the user a warning if the release time isn't a multiple of outputdt."""
-    if any(t % outputdt != 0 for t in release_times):
+    if any((np.isfinite(t) and t % outputdt != 0) for t in release_times):
```

Comment:
Is this change required? If `t = np.nan` then `t % outputdt != 0` would always be False anyway.

Reply:
Well, without this extra statement the Parcels tutorial gives these warnings whenever the time is not set
<img width="911" alt="Screenshot 2024-10-28 at 08 37 01" src="https://github.com/user-attachments/assets/309121fb-f968-4b33-b633-7ede88e17390">

Reply:
👍  :)